external-tracer/src/lib.rs

```diff
@@ -23,6 +23,10 @@ pub struct TraceConfig {
     pub transactions: Vec<Transaction>,
     /// logger
     pub logger_config: LoggerConfig,
+    /// taiko
+    pub taiko: bool,
```

Comment:
taiko as name is confusing, consider change

Reply:
the `taiko` is a flag in taiko-geth's ChainConfig